src/Controls/src/Core/Menu/MenuItem.cs

```diff
@@ -145,6 +145,9 @@ static object CoerceIsEnabledProperty(BindableObject bindable, object value)
 			return false;
 		}
 
+		IReadOnlyList<IAccelerator> IMenuElement.Accelerators =>
+			GetAccelerator(this) is Accelerator acc ? new[] { acc } : null;
```

Comment:
The order matters I think because you want the accelerators to be in a specific order. The null is not my favorite but some people like it - not sure it matters. We can always not-null it later, but making it null later is breaking.